internal/app/app.go

```diff
@@ -79,11 +79,18 @@ type FileChange struct {
 	Patch       string `json:"patch"`
 }
 
+// PRReviewMap is a map of file paths to a list of PRCommentInfo
 type PRReviewMap map[string][]PRCommentInfo
 type PRCommentInfo struct {
 	CommentBody string
-	StartLine   int
-	Line        int
+	// The start_line is the first line in the pull request diff that your multi-line comment applies to.
+	StartLine int
+	// The line of the blob in the pull request diff that the comment applies to. For a multi-line comment, the last line of the range that your comment applies to.
```

Comment:
For the added `Side` and `StartSide` fields in the `PRCommentInfo` struct, consider adding enum validations if applicable to restrict the values to only 'LEFT' or 'RIGHT'.